scenariolib/visit.go

```diff
@@ -473,7 +473,8 @@ func WaitBetweenActions(timeToWait int, isConstant bool) {
 	if isConstant {
 		time.Sleep(time.Duration(timeToWait) * time.Second)
```

Comment:
I guess yes I was too permissive and thought that if you set a constant time duration you would know what you are doing :D